db/src/db.rs

```diff
@@ -731,14 +732,19 @@ impl DB {
             .chain_err(|| "Could not update datoms: failed to retract datoms already present")?;
 
         // Insert datoms that were added and not already present.
-        let s = r#"
+        let s = format!(r#"
```

Comment:
This is performance benchmarking, so someone should have an eye on this query other than me